### ISSUE TITLE
Cannot use group default action on an empty group.

### ISSUE THREAD

**Issue by db0** [octgn/OCTGN#1521]

**Comment by Soul1355** [comment]
yeah, I think off the top of my head that the event is captured by the card in the pile (which is basically the same as the ones on the table) so it might take a bit of work to move that over to the pile itself. might just be able to add a trigger in the blank space, but that depends. 
just as idle curiosity what did you have in mind for this?

**Comment by brine** [comment]
yea my first thought here was that the default action only worked for cardactions?

**Comment by db0** [comment]
I have a pile with a default action which I want to A) Draw cards when the pile is empty or B) Place cards on the table when the pile has cards in it.
